deployments/container/reconfigure-mig.sh

```diff
@@ -111,7 +115,13 @@ while getopts "hrden:f:c:m:i:o:g:k:p:t:a:b:j:" opt; do
     j ) # process option j
       DEV_ROOT_CTR_PATH=${OPTARG}
       ;;
-    \? ) echo "Usage: ${0} -n <node> -f <config-file> -c <selected-config> -p <default-gpu-clients-namespace> [-e -t <driver-root> -a <driver-root-ctr-path> -b <dev-root> -j <dev-root-ctr-path>] [ -m <host-root-mount> -i <host-nvidia-dir> -o <host-mig-manager-state-file> -g <host-gpu-client-services> -k <host-kubelet-service> -r -s ]"
+    l ) # process option l
+      DRIVER_LIBRARY_PATH=${OPTARG}
+      ;;
+    q ) # process option q
+      NVIDIA_SMI_PATH=${OPTARG}
+      ;;
+    \? ) echo "Usage: ${0} -n <node> -f <config-file> -c <selected-config> -p <default-gpu-clients-namespace> [-e -t <driver-root> -a <driver-root-ctr-path> -b <dev-root> -j <dev-root-ctr-path> -l <driver-library-path> -q <nvidia-smi-path> ] [ -m <host-root-mount> -i <host-nvidia-dir> -o <host-mig-manager-state-file> -g <host-gpu-client-services> -k <host-kubelet-service> -r -s ]"
```

Comment:
Any reason why we don't call `usage` and exit here? (like we do for `h`)

Reply:
I am not aware of any reason not to. I can update this to call `usage` and `exit` in a follow-up.